src/main/resources/application.yml

```
@@ -2,7 +2,7 @@ spring:
   application:
     name: techchallenge
   profiles:
-    active: ${SPRING_PROFILES_ACTIVE:default}
+    active: ${SPRING_PROFILES_ACTIVE:mem}
```

Deixar o valor padrão como `default`. (Quando for executar em memória, adicionar uma variável de ambiente `SPRING_PROFILES_ACTIVE` com o valor `mem`)
```suggestion
    active: ${SPRING_PROFILES_ACTIVE:default}
```